pom.xml

```diff
@@ -14,20 +14,39 @@
   </licenses>
 
   <properties>
-    <aspectj.version>1.9.19</aspectj.version>
-    <raml-module-builder.version>35.0.1</raml-module-builder.version>
-    <vertx.version>4.3.4</vertx.version>
-    <junit.jupiter.version>5.8.2</junit.jupiter.version>
-    <rest-assured.version>5.1.1</rest-assured.version>
-    <log4j-slf4j-impl.version>2.19.0</log4j-slf4j-impl.version>
-    <folio-di-support.version>1.6.0</folio-di-support.version>
-    <wiremock.version>2.26.3</wiremock.version>
-    <jsonschema_paths>acq-models/mod-orgs/schemas,raml-util/schemas,acq-models/common/schemas,acq-models/acquisitions-unit/schemas</jsonschema_paths>
+    <!--Project properties-->
+    <java.version>17</java.version>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <generated_sources_dir>${project.build.directory}/generated-sources</generated_sources_dir>
     <jsonschema2pojo_output_dir>${generated_sources_dir}/jsonschema2pojo</jsonschema2pojo_output_dir>
-    <log4j.version>2.17.2</log4j.version>
+    <jsonschema_paths>acq-models/mod-orgs/schemas,raml-util/schemas,acq-models/common/schemas,acq-models/acquisitions-unit/schemas</jsonschema_paths>
+
+    <!--Dependency Management Properties-->
+    <log4j.version>2.20.0</log4j.version>
+
+    <!--Dependency properties-->
+    <aspectj.version>1.9.20.1</aspectj.version>
+    <raml-module-builder.version>35.1.0</raml-module-builder.version>
+    <vertx.version>4.4.5</vertx.version>
+    <junit.jupiter.version>5.10.0</junit.jupiter.version>
+    <rest-assured.version>5.3.2</rest-assured.version>
+    <folio-di-support.version>2.0.1</folio-di-support.version>
+    <wiremock.version>2.26.3</wiremock.version>
+
+    <!--Maven plugin properties-->
+    <aspectj-maven-plugin.version>1.13.1</aspectj-maven-plugin.version>
+    <exec-maven-plugin.version>3.1.0</exec-maven-plugin.version>
+    <maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
+    <maven-compiler-plugin.version>3.11.0</maven-compiler-plugin.version>
+    <maven-antrun-plugin.version>3.1.0</maven-antrun-plugin.version>
+    <maven-shade-plugin.version>3.5.1</maven-shade-plugin.version>
+    <properties-maven-plugin.version>1.0.0</properties-maven-plugin.version>
```

Comment:
In the latest tag of RMB version of properties-maven-plugin is 1.2.0, so lets use the same
https://github.com/folio-org/raml-module-builder/blob/v35.1.1/pom.xml#L342

Reply:
Done